frontend/cypress/tests/codeView_load_benchmark.js

```diff
@@ -0,0 +1,34 @@
+describe('load code view benchmark', function() {
+  const MAXIMUM_LOADING_TIME = 8000;
```

Comment:
How did you determine the threshold of 8 seconds?

The current arguments used for Cypress test is `--repo https://github.com/reposense/RepoSense.git --view` which will analyse the repository from one month from today's date until today. In the future if the author Eugene makes a lower contribution compared to this month, even if the code view loading speed has decreased, this benchmark test will not be able to detect it as the author Eugene has lower contribution. 

Unless we set the arguments used for Cypress test to be within a constant duration, using a constant threshold may not be a good idea for this benchmark test.